### DISCUSSION TITLE
electron phonon scattering contribution to lattice thermal conductivity

### DISCUSSION THREAD

**Discussion by akentphonon** [phoebe-team/phoebe#135] [Q&A] [answered]
Dear developers,

Thank you for this nice tool.

A quick query:

Is it possible to include contribution from electron-phonon scattering to lattice thermal conductivity in phoebe?

Thanks.
AK

**Comment by cepellotti** [comment, marked as answer]
Hi AK,
sorry for the late reply (somehow my notifications from this thread were turned off).

To your point: no, it's not implemented right now. It's on our to-do list though, since it would be relatively easy to add.

Best,
Andrea
